justfile

```diff
@@ -52,7 +52,7 @@ test-e2e testname:
 
 # Install the sp1-ics07-tendermint operator for use in the e2e tests
 install-operator:
-	cargo install --git https://github.com/cosmos/sp1-ics07-tendermint --rev {{sp1_operator_rev}} sp1-ics07-tendermint-operator --bin operator --locked
+	cargo +nightly install --git https://github.com/cosmos/sp1-ics07-tendermint --rev {{sp1_operator_rev}} sp1-ics07-tendermint-operator --bin operator --locked
```

Comment:
i had to do this to get it to install on my machine. Can remove if desired

Reply:
Ah yes, makes sense. The most recent operator commit doesn't need this since we removed the union dependencies